gapic/src/main/com/google/gapid/perfetto/models/FrameEventsTrack.java

```diff
@@ -205,14 +241,14 @@ public Data(DataRequest request, long bucketSize, String[] concatedIds, long[] n
       this.ids = null;
       this.starts = null;
       this.ends = null;
-      this.depths = null;
```

Comment:
The depth is still queried as one of the base columns in the SQL query, and we also still keep the getSlices method inside this file using depth as parameter.. Do we want to remove that as well?

Reply:
My bad. I removed the depth from the query as well, but I think that got lost while cleaning up. I'll remove the depth from the queries altogether.

Reply:
Just checked again, depth is being used in the span_join operator and removing it from the query is going to change a lot of things. I suggest just removing depths from the data for now, since we are not using it.